x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/moving_average.tsx

```diff
@@ -168,3 +175,51 @@ function MovingAverageParamEditor({
     </EuiFormRow>
   );
 }
+
+const MovingAveragePopup = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  return (
+    <EuiPopover
+      ownFocus
+      isOpen={isPopoverOpen}
+      button={
+        <EuiText size="xs">
+          <EuiLink onClick={() => setIsPopoverOpen(!isPopoverOpen)}>
+            {i18n.translate('xpack.lens.indexPattern.movingAverage.helpText', {
+              defaultMessage: 'How does it work?',
+            })}
+          </EuiLink>
+        </EuiText>
+      }
+      closePopover={() => setIsPopoverOpen(false)}
+      anchorPosition="leftCenter"
+    >
+      <EuiPopoverTitle>
+        <EuiIcon type="help" />
+        &nbsp;{' '}
+        {i18n.translate('xpack.lens.indexPattern.movingAverage.titleHelp', {
+          defaultMessage: 'How does moving average work?',
+        })}
+      </EuiPopoverTitle>
+      <EuiText size="s" style={{ width: 300 }}>
+        <p>
+          {i18n.translate('xpack.lens.indexPattern.movingAverage.basicExplanation', {
+            defaultMessage:
+              'The Moving Average slides a window across the data and emits the average value of that window.',
+          })}
+        </p>
+        <p>
+          {i18n.translate('xpack.lens.indexPattern.movingAverage.longerExplanation', {
+            defaultMessage: `The Lens Moving Average uses a simple arithmetic mean of the window and applies a skip policy for gaps: 
+                this means that for missing values the bucket is skipped and the calculation is performed on the next one.`,
```

Comment:
Would this be easier to explain if we showed an example in table form?

Reply:
Added an example to solve this.